packages/shared/src/components/tooltips/TooltipContainer.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactElement, ReactNode } from 'react';
+import classNames from 'classnames';
+
+export type TooltipPosition = 'top' | 'right' | 'bottom' | 'left';
+
+export interface TooltipContainerProps {
+  arrow?: boolean;
+  children: ReactNode;
+  className?: string;
+  arrowClassName?: string;
+  placement?: TooltipPosition;
+}
+
+export function TooltipContainer({
+  className,
+  arrow = true,
+  arrowClassName,
+  placement = 'top',
+  children,
+}: TooltipContainerProps): ReactElement {
+  return (
+    <div
+      data-popper-placement={placement}
+      className={classNames(
+        'relative flex items-center py-1 px-3 rounded-10 bg-theme-label-primary text-theme-label-invert typo-subhead',
```

Comment:
Just a question, is the flex-row direction ok here, as we haven't defined it specifically?
Just making sure the items should show next to each other.

Reply:
Great question, yes, I think we should make this predictable by using either `flex-row` or `flex-col`.